frontend/src/pages/ErrorsV2/ErrorInstances/ErrorInstancesTable.tsx

```diff
@@ -78,7 +78,7 @@ export const ErrorInstancesTable = ({ edges, searchedEmail }: Props) => {
 				let content = <>no session</>
 				let sessionLink = ''
 
-				if (session) {
+				if (session && !session.excluded) {
```

Comment:
Currently just showing the same message as if the session is not present, but maybe we want to display a different message. Not sure all the use cases here, but could be a callout to start recording sessions?